Avoid build error seen on jammy and noble:

  * Building wheel...
  Traceback (most recent call last):
    File "/usr/lib/python3/dist-packages/pyproject_hooks/_in_process/_in_process.py", line 353, in <module>
      main()
    File "/usr/lib/python3/dist-packages/pyproject_hooks/_in_process/_in_process.py", line 335, in main
      json_out['return_val'] = hook(**hook_input['kwargs'])
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    File "/usr/lib/python3/dist-packages/pyproject_hooks/_in_process/_in_process.py", line 251, in build_wheel
      return _build_backend().build_wheel(wheel_directory, config_settings,
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    File "/usr/lib/python3/dist-packages/hatchling/build.py", line 58, in build_wheel
      return os.path.basename(next(builder.build(directory=wheel_directory, versions=['standard'])))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    File "/usr/lib/python3/dist-packages/hatchling/builders/plugin/interface.py", line 90, in build
      self.metadata.validate_fields()
    File "/usr/lib/python3/dist-packages/hatchling/metadata/core.py", line 244, in validate_fields
      self.core.validate_fields()
    File "/usr/lib/python3/dist-packages/hatchling/metadata/core.py", line 1334, in validate_fields
      getattr(self, attribute)
    File "/usr/lib/python3/dist-packages/hatchling/metadata/core.py", line 728, in license_files
      raise TypeError(message)
  TypeError: Field `project.license-files` must be a table

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -22,8 +22,7 @@ keywords = [
     "sponsorblock",
     "yt-dlp",
 ]
-license = "Unlicense"
-license-files = ["LICENSE"]
+license = {file = "LICENSE"}
 classifiers = [
     "Topic :: Multimedia :: Video",
     "Development Status :: 5 - Production/Stable",
